extensions/tailscale/raycast-env.d.ts

```diff
@@ -40,3 +40,13 @@ declare namespace Arguments {
   /** Arguments passed to the `admin` command */
   export type Admin = {}
 }
+
+
+declare module "swift:*" {
```

Comment:
This was automatically added, I assume by the latest version of the Raycast toolkit.

Reply:
Yes, it's autogenerated and should actually be added to the gitignore file 🙂

Reply:
Oh, good to know! I'll include that with this change too.